src/Microsoft.Diagnostics.Monitoring.WebApi/StreamingLogger.cs

```diff
@@ -47,6 +47,9 @@ public sealed class StreamingLogger : ILogger
         private readonly LogFormat _logFormat;
         private readonly LogLevel? _logLevel;
 
+        private static readonly int LogTextPaddingLength = GetLogLevelString(LogLevel.Information).Length + ": ".Length;
```

Comment:
Maybe put a comment? I'm guessing you want the longest possible log level padding?

Reply:
It's the same padding that json console logger uses. I'll add a comment about it though.